components/tabs/tabs.js

```diff
@@ -1015,6 +997,26 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 		return document.documentElement.getAttribute('dir') === 'rtl';
 	}
 
+	async #tabSelectionHelper(selectedTab) {
```

Comment:
If it makes more sense, maybe this could be named `#updateSelectedTab` or `#updateSelectedTabState`.